code/modules/jobs/job_types/cook.dm

```diff
@@ -54,6 +54,17 @@
 
 		kitchen_areas |= path
 
+	mail_goodies = list(
+		/obj/item/storage/box/ingredients/random = 80,
+		/datum/reagent/consumable/caramel =  20,
+		/obj/item/reagent_containers/food/condiment/flour = 20,
+		/obj/item/reagent_containers/food/condiment/rice = 20,
+		/obj/item/reagent_containers/food/condiment/enzyme = 15,
+		/obj/item/reagent_containers/food/condiment/soymilk = 15,
```

Comment:
would a live chicken or something be off the table here?

Reply:
Chicken cubes, like monkey cubes, but taste like chicken.